tss-esapi/src/tcti_ldr.rs

```diff
@@ -247,6 +256,16 @@ impl FromStr for TctiNameConf {
             )?));
         }
 
+        let libtpms_pattern = Regex::new(r"^libtpms(:(.*))?$").unwrap(); //should not fail
+        if let Some(captures) = libtpms_pattern.captures(config_str) {
+            return Ok(TctiNameConf::LibTpms {
+                state: captures
+                    .get(2)
+                    .map(|m| m.as_str())
+                    .and_then(|s| PathBuf::from_str(s).ok()),
```

Comment:
```suggestion
                    .and_then(|m| PathBuf::from_str(m.as_str()).ok()),
```

Well, maybe it's just personal taste :sweat_smile: